arbeitszeiterfassung_s4a/arbeitszeiterfassung_s4a/doctype/working_time/working_time.js

```diff
@@ -27,6 +27,10 @@ frappe.ui.form.on("Working Time Log", {
 		let prev_to_time;
 		if (frm.doc.time_logs.length > 1) {
 			prev_to_time = frm.doc.time_logs.at(-2).to_time;
+			prev_row = frm.doc.time_logs.at(-2).name;
+			if (!prev_to_time || prev_to_time.trim() === "") {
+				frappe.model.set_value(cdt, prev_row, "to_time", frappe.datetime.now_time(false));
+			  }
 		}
```

Comment:
```suggestion
		if (frm.doc.time_logs.length > 1) {
			prev_row = frm.doc.time_logs.at(-2);
			if (!prev_row.to_time || prev_row.to_time.trim() === "") {
				frappe.model.set_value(cdt, prev_row.name, "to_time", frappe.datetime.now_time(false));
			}
		}
```

So können wir uns eine Variable sparen und es ist klarer, wo die Daten herkommen

Reply:
@barredterra 
Danke.
Habe ich so berücksichtigt mit kleinen Änderungen (Variable oberhalb des If-Statements und prev_to_time an anderen Stellen ersetzt).